states/AL/governor.yaml

```diff
@@ -0,0 +1,108 @@
+# Max length is not used at this time.  However, unitedstates/contact-congress has an option for it, so it is included in this form as a comment in case there is a desire to implement it in the future.
+
+contact_form:
+  steps:
+    - visit: "https://contact.governor.alabama.gov/contact.aspx"
+    - select:
+        - name: "ctl00$ContentPlaceHolder1$PREFIX"
+          selector: "#ContentPlaceHolder1_PREFIX"
+          value: $NAME_PREFIX
+          required: true
+          options:
+            "Mr.": "Mr."
+            "Ms.": "Ms."
+            "Mrs.": "Mrs."
+            "Dr.": "Dr."
+    - fill_in:
+        - name: "ctl00$ContentPlaceHolder1$FIRSTNAME"
+          selector: "#ContentPlaceHolder1_FIRSTNAME"
+          value: $NAME_FIRST
+          required: true
+          # max length: 60
+        - name: "ctl00$ContentPlaceHolder1$LASTNAME"
+          selector: "#ContentPlaceHolder1_LASTNAME"
+          value: $NAME_LAST
+          required: true
+          # max length: 60
+        - name: "ctl00$ContentPlaceHolder1$EMAILADDRESS"
+          selector: "#ContentPlaceHolder1_EMAILADDRESS"
+          value: $EMAIL
+          required: true
+          # max length: 100
+        - name: "ctl00$ContentPlaceHolder1$PHONE1"
+          selector: "#ContentPlaceHolder1_PHONE1"
+          value: $PHONE_AREA_CODE
+          required: true
+          # max length: 3
+        - name: "ctl00$ContentPlaceHolder1$PHONE2"
+          selector: "#ContentPlaceHolder1_PHONE2"
+          value: $PHONE3
+          required: true
+          # max length: 3
+        - name: "ctl00$ContentPlaceHolder1$PHONE3"
+          selector: "#ContentPlaceHolder1_PHONE3"
+          value: $PHONE4
+          required: true
+          # max length: 4
+        - name: "ctl00$ContentPlaceHolder1$STREET_ADDRESS"
+          selector: "#ContentPlaceHolder1_STREET_ADDRESS"
+          value: $ADDRESS_STREET
+          required: true
+          # max length: 200
+        - name: "ctl00$ContentPlaceHolder1$CITY"
+          selector: "#ContentPlaceHolder1_CITY"
+          value: $ADDRESS_CITY
+          required: true
+          # max length: 100
+        - name: "ctl00$ContentPlaceHolder1$STATE"
+          selector: "#ContentPlaceHolder1_STATE"
+          value: $ADDRESS_STATE_FULL
```

Comment:
Should probably be $ADDRESS_STATE_POSTAL_ABBREV?

Reply:
The form accepts a full state name.  Is it preferred to always use the abbreviation, even when not limited to 2 characters?

Reply:
Hehe yeah, canonical postal address format in all contexts in the US is with state postal code eh?

Reply:
Doesn't matter to me either way.  I guess I was just thinking about how I would personally fill out the form.  I'll change it.